src/components/projectDetail/DonateDock.tsx

```
@@ -645,273 +694,324 @@ const DonateDock = () => {
+                                    <div
+                                      key={optionId}
+                                      className="border border-gray-border rounded-lg overflow-hidden"
+                                    >
+                                      {/* 상품 헤더 - 클릭 시 펼침/접힘 */}
+                                      <div
+                                        className="flex justify-between items-center p-4 cursor-pointer hover:bg-gray-50"
+                                        onClick={(e) => toggleProductExpand(optionId, e)}
+                                      >
+                                        <div className="flex items-center">
+                                          <ChevronRight
+                                            className={`h-5 w-5 mr-2 text-sub-gray transition-transform ${
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)